shell/common/shell.cc

```diff
@@ -1254,4 +1258,10 @@ fml::Status Shell::WaitForFirstFrame(fml::TimeDelta timeout) {
   }
 }
 
+void Shell::SetFirstFrameCallback(std::function<void()> callback) {
```

Comment:
I think having separate SetFirstFrameCallback and OnPlatformViewSetNextFrameCallback is ok as discussed offline. Though we should be consistent about both using platform views or not.